src/overlays/actors/ovl_Dm_Char01/z_dm_char01.c

```diff
@@ -28,44 +46,498 @@ const ActorInit Dm_Char01_InitVars = {
     (ActorFunc)DmChar01_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80AAAAB0[] = {
+static InitChainEntry sInitChain[] = {
     ICHAIN_F32(uncullZoneScale, 300, ICHAIN_STOP),
 };
 
-#endif
+static s16 D_80AAAAB4 = 0;
```

Comment:
```suggestion
static s16 D_80AAAAB4 = false;
```
This variable is boolean, you can change throughout